### ISSUE TITLE
Fix logs compliance with ESC (errors and hash fields)

### ISSUE THREAD

**Issue by juliocamarero** [elastic/fleet-server#358]
In `7.13` the logs claim to be ECS compliant but these two fields are not following ECS:

```json
"ecs": {
   "version": "1.8.0"
},
"error": "context canceled",
"hash": "81b3d070dddec145fafcbdfb6f22888495a12edc31881f6b0511fa10de66daa7",
```

The `error` according to ECE 1.8.0 should be an object, not a string ([docs](https://www.elastic.co/guide/en/ecs/1.8/ecs-error.html)). The field `hash` should also be an object ([docs](https://www.elastic.co/guide/en/ecs/1.8/ecs-hash.html)).

cc @scunningham

**Comment by ph** [comment]
@juliocamarero Can you give us the complete document? I want to make sure we have the correct source.

**Comment by juliocamarero** [comment]
Sure @ph, here are the logs [logs.tar.gz](https://github.com/elastic/fleet-server/files/6494190/logs.tar.gz)

You can also see this other issue in those: https://github.com/elastic/fleet-server/issues/305

**Comment by EricDavisX** [comment]
how do we check for log compliance at dev time / automated tests?  we may have tools i'm unaware of, would like to know more.

**Comment by EricDavisX** [comment]
> how do we check for log compliance at dev time / automated tests? we may have tools i'm unaware of, would like to know more.

**Comment by EricDavisX** [comment]
Actually - I'm reviewing and we have a fix backported to 7.13... it will be in the build by tonight, so testable as a snapshot in cloud-staging or on cloud-qa (I think?) tomorrow, 5/18 - let's close it out to reduce the blocker's list for clarity's sake
